client-app/src/Components/Register.js

```diff
@@ -32,146 +33,117 @@ const Register = () => {
             setErrorMessage('Password must be at least 5 characters');
             return;
         }
-
```

Comment:
Can you also check why others changes are appearing in your PR? As you have already merged the upstream main branch into your working branch, these changes of others are not supposed to be shown here.